lib/actions/actionsBuilder.js

```diff
@@ -0,0 +1,30 @@
+const AddLinkToProjectAction = require('./AddLinkToProjectAction');
+const assert = require('assert');
+
+class ActionsBuilder {
```

Comment:
Personally, I would rename this to `Actions`. Remove the "pattern" name from the class. Convey the intent, not how it's constructed.